nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-services/src/main/java/org/apache/nifi/mongodb/MongoDBControllerService.java

```diff
@@ -105,7 +110,18 @@ public final void closeClient() {
     }
 
     protected String getURI(final ConfigurationContext context) {
-        return context.getProperty(URI).evaluateAttributeExpressions().getValue();
+        final String _uri = context.getProperty(URI).evaluateAttributeExpressions().getValue();
```

Comment:
Stylish comment: not sure the use of an underscore is very common in the NiFi codebase, I'd just go for ``uri``.

Reply:
My intent was an "in construction" variable to be assigned later to "uri", but I changed my mind and I put direct to return. I'll change it.